proto/rollkit/rollkit.proto

```diff
@@ -53,8 +53,11 @@ message Header {
 	// Hash of block aggregator set, at a time of block creation
 	bytes aggregators_hash = 11;
 
+	// Hash of next block aggregator set, at a time of block creation
+	bytes next_aggregators_hash = 12;
+
 	// Chain ID the block belongs to
-	string chain_id = 12;
+	string chain_id = 13;
```

Comment:
At some point, we should start using protobufs fields IDs [as indented](https://protobuf.dev/programming-guides/proto3/#assigning-field-numbers) for compatibility reasons, instead of modifying them all the time.

Reply:
Noted.